environments/gcp/iam.tf

```diff
@@ -0,0 +1,64 @@
+resource "google_iam_workload_identity_pool" "workload-identity-pool" {
+  workload_identity_pool_id = "${local.project_id}"
+}
+#add this
+
+
+resource "google_service_account" "flyte_binary" {
+  account_id = "${local.name_prefix}-flyte-binary"
+}
+
+data "google_iam_policy" "flyte_binary_workload_identity" {
```

Comment:
Thought:
In GCP, IAM policies are very authorative and one can easily take away permissions by other members by accident. Especially dangerous on the project level where one can easily [lock onself out](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project_iam#google_project_iam_policy) 😆 

Since this is not on the project level, it might not be thaat bad but I personally would have created multiple [`google_service_account_iam_member`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_service_account_iam#google_service_account_iam_member) resources instead.

Reply:
Feel free to just ignore/resolve!